history-of-quakers/modernized/18-chapter-16.adoc

```diff
@@ -2,21 +2,40 @@
 
 === 1664-1667.
 
-Continuation of persecution under the Conventicle Act--Account of the Author`'s Parents--
-"`Epistle to the Friends of Truth`"--Six hundred Friends in the jails of England--some
-of those banished set ashore by the Captains of Wessels--Wessels not allowed to sail
-to the West Indies without taking banished Quakers--Plague in London -Further trials
-and banishment of Quakers--I. Penington on Persecution--G. Fox and Margaret Fell again
-brought to trial and premunired--Removal of G. Fox to Scarborough Castle-- Interesting
-interviews--Incidents in the life of W. Dundas--Death of W. Caton--Death of many banished
-Friends on board ship -- Captured by a Privateer--G. Fox and Dr. Cradock--G. Fox released
--- Great fire in London--T. Ibbitt -- Epistle of S. Crisp - Persecution by W. Armorer--Death
-of R. Farnsworth--Robert Barclay-Persecutors of M. Halhead.
+[.chapter-synopsis]
+* Continuation of persecution under the Conventicle Act
+* Account of the Author`'s Parents
+* "`Epistle to the Friends of Truth`"
+* Six hundred Friends in the jails of England
+* Some of those banished set ashore by the Captains of Vessels
+* Vessels not allowed to sail to the West Indies without taking banished Quakers
+* Plague in London
+* Further trials and banishment of Quakers
+* I. Penington on Persecution
+* G. Fox and Margaret Fell again brought to trial and premunired
+* Removal of G. Fox to Scarborough Castle
+* Interesting interviews
+* Incidents in the life of W. Dundas
+* Death of W. Caton
+* Death of many banished Friends on board ship
+* Captured by a Privateer
+* G. Fox and Dr. Cradock
+* G. Fox released
+* Great fire in London
+* T. Ibbitt
+* Epistle of S. Crisp
+* Persecution by W. Armorer
+* Death of R. Farnsworth
+* Robert Barclay
+* Persecutors of M. Halhead
 
 Again, in December, twelve of those called Quakers, were condemned to banishment;
 for it was resolved to go on, not regarding what George Bishop gave forth in print,
 and caused to be delivered to the king and the members of parliament, being as follows:
 
+[.embedded-content-document.address]
+--
+
 To the King and both Houses of Parliament, this says the Lord.
```

Comment:
Should this be a [.letter-head]? It looks a little funny just as plain text, but doesn't seem to qualify as a true letter head to me. Idk.

Reply:
Yeah, that's super weird. I think it would be better as a [.letter-heading], even though it makes me feel a little funny.

Also, can you change `this` to `thus`? It looks like that's a scan error.